src/blog/2024/04/building-an-admin-panel-in-node-red-with-dashboard-2.md

```diff
@@ -0,0 +1,111 @@
+---
+title: How to Build an Admin Dashboard with Node-RED Dashboard 2.0
+subtitle: A guide to building an Admin Panel in Node-RED with Dashboard 2.0
+description: Discover step-by-step instructions for developing an admin-only page in Node-RED Dashboard 2.0 using the FlowFuse Multiuser addon. Additionally, learn how to secure Dashboard 2.0 and explore the features of the FlowFuse multiuser addon.
+date: 2024-04-05
+authors: ["sumit-shinde"]
+image: 
+tags:
+   - posts
+   - dashboard 2.0
+
+---
+
+As businesses grow, data doubles, and managing and analyzing such increasing amounts of data becomes crucial. businesses are using Node-RED and its Dashboard 2.0 to handle this effectively. But what about controlling who can access what data? That's where an admin-only page comes in. Now With Node-RED Dashboard 2.0, we can also create robust and secure admin-only pages easily. In this guide, we'll provide you with step-by-step instructions to Build an Admin-only page with Node-RED Dashboard 2.0.
```

Comment:
@sumitshinde-84 statements like "data doubles" don't add anything here for me and are too conversational. Please refrain from such text going forward until we want to change the tone on our blog posts.

```suggestion
Managing and analyzing increasing amounts of data becomes crucial for organisations. Dashboard 2.0 and Node-RED help organisations to access the data, normalize it, and visualize it. But what about controlling who can access what data? That's where an admin-only page comes in. Now With Node-RED Dashboard 2.0, we can also create robust and secure admin-only pages easily. In this guide, we'll provide you with step-by-step instructions to Build an Admin-only page with Node-RED Dashboard 2.0.
```